I.pass-tests/07-second-largest/second-largest.js

```diff
@@ -0,0 +1,8 @@
+function secondLargest(array) {
+    let actualMax = Math.max(...array)
+    let maxIndex = array.indexOf(actualMax)
+    array.splice(maxIndex, 1)
+    return Math.max(...array)
+}
```

Comment:
nice work, and works fine
the only a couple of issues, the complexity is high here 
You needed to loop through the array twice to get the max 
also what if max number is duplicated ?